tests/rptest/services/redpanda.py

```diff
@@ -273,7 +273,7 @@ class RedpandaService(Service):
                                             "redpanda.profraw")
 
     CLUSTER_NAME = "my_cluster"
-    READY_TIMEOUT_SEC = 10
+    READY_TIMEOUT_SEC = 30
```

Comment:
`ducky: increase start up timeout`

can you add a note in the commit message about what may have been taking so long to start up the broker?

a few days ago this was even reduced from 15 to 10 seconds. a slow start-up could be an indication of a deeper issue / regression, and may even treated as a failure.

Reply:
it's irrelevant to this pr, I dropped this commit